components/content_settings/core/browser/brave_content_settings_pref_provider.h

```diff
@@ -35,10 +40,30 @@ class BravePrefProvider : public PrefProvider {
       ContentSettingsType content_type,
       const ResourceIdentifier& resource_identifier,
       std::unique_ptr<base::Value>&& value) override;
+  std::unique_ptr<RuleIterator> GetRuleIterator(
+      ContentSettingsType content_type,
+      const ResourceIdentifier& resource_identifier,
+      bool incognito) const override;
+
+ private:
+  void UpdateCookieRules(ContentSettingsType content_type, bool incognito);
+  void OnCookieSettingsChanged(ContentSettingsType content_type);
+
+  // content_settings::Observer overrides:
+  void OnContentSettingChanged(const ContentSettingsPattern& primary_pattern,
+                               const ContentSettingsPattern& secondary_pattern,
+                               ContentSettingsType content_type,
+                               const std::string& resource_identifier) override;
+  void NotifyChanges(const std::vector<Rule>& rules, const bool incognito);
```

Comment:
const looks useless?

Reply:
and this method doesn't override anything, so we can move it, or separate with a blank line